langfuse-core/src/types.ts

```diff
@@ -127,8 +127,8 @@ export type TextPrompt = FixTypes<components["schemas"]["TextPrompt"]> & { type:
 
 type CreateTextPromptRequest = FixTypes<components["schemas"]["CreateTextPromptRequest"]>;
 type CreateChatPromptRequest = FixTypes<components["schemas"]["CreateChatPromptRequest"]>;
-export type CreateTextPromptBody = { type?: "text" } & Omit<CreateTextPromptRequest, "type">;
-export type CreateChatPromptBody = { type: "chat" } & Omit<CreateChatPromptRequest, "type">;
+export type CreateTextPromptBody = { type?: "text" } & Omit<CreateTextPromptRequest, "type"> & { isActive?: boolean }; // isActive is optional for backward compatibility
```

Comment:
did you add these manually? If yes, the code in here should be auto-generated i think.